### PR TITLE
[community]: Render documents to graphviz

### PR DESCRIPTION
**Pull request by bjchambers**
- **Description:** Adds a helper that renders documents with the GraphVectorStore metadata fields to Graphviz for visualization. This is helpful for understanding and debugging.

### CODE REVIEW

**Comment by bjchambers** [comment]
@cbornet

File: libs/community/langchain_community/graph_vectorstores/visualize.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        return s
+    else:
+        return f"{s[0:split]}..."
+
+
+def render_graphviz(
+    documents: Iterable[Document],
+    engine: Optional[str] = None,
+    node_color: Optional[str] = None,
+    node_colors: Optional[Dict[str, Optional[str]]] = {},
```
Comment (cbornet) [inline libs/community/langchain_community/graph_vectorstores/visualize.py:43]: Should not assign a mutable as default value.

Comment (cbornet) [inline libs/community/langchain_community/graph_vectorstores/visualize.py:43]:
```suggestion
    node_colors: Optional[Dict[str, Optional[str]]] = None,
```

**Comment by bjchambers** [comment]
@eyurtsev This PR adds a method for rendering the graph, making it easier for users to understand how things operate and debug issues. Would it be possible to get this merged?

**Review by efriis — APPROVED** [review]
made it beta to match abstraction

**Pull request merged**